cmd/seednode/main.go

```diff
@@ -242,13 +242,14 @@ func loadMainConfig(filepath string) (*config.Config, error) {
 
 func createNode(p2pConfig config.P2PConfig, marshalizer marshal.Marshalizer) (p2p.Messenger, error) {
 	arg := libp2p.ArgsNetworkMessenger{
-		Marshalizer:          marshalizer,
-		ListenAddress:        libp2p.ListenAddrWithIp4AndTcp,
-		P2pConfig:            p2pConfig,
-		SyncTimer:            &libp2p.LocalSyncTimer{},
-		PreferredPeersHolder: disabled.NewPreferredPeersHolder(),
-		NodeOperationMode:    p2p.NormalOperation,
-		PeersRatingHandler:   disabled.NewDisabledPeersRatingHandler(),
+		Marshalizer:           marshalizer,
+		ListenAddress:         libp2p.ListenAddrWithIp4AndTcp,
+		P2pConfig:             p2pConfig,
+		SyncTimer:             &libp2p.LocalSyncTimer{},
+		PreferredPeersHolder:  disabled.NewPreferredPeersHolder(),
+		NodeOperationMode:     p2p.NormalOperation,
+		PeersRatingHandler:    disabled.NewDisabledPeersRatingHandler(),
+		ConnectionWatcherType: "disabled",
```

Comment:
that flag hasn't been altered in the previous code. also, might use constants

Reply:
added prefs.toml to seednode

Reply:
one more config for the seednode :( 
Since it won't work on the seednode binary anyway, I suggest to leave it as a const.